packages/test/test-version-utils/src/baseVersion.ts

```diff
@@ -22,10 +22,19 @@ function getCodeVersion() {
 	return pkgVersion;
 }
 
-// So for test branches, the base version is 0.0.0-xyz-test where xyz is the build number.
-// So if we use the base version we will not get the right back compat testing and thus we want to use the code version
+/**
+ * So for test branches, the base version is 0.0.0-xyz-test where xyz is the build number.
```

Comment:
These comments aren't describing what this variable is. I'm not really sure what the context of this documentation is, but I don't think it belongs here. I would maybe put this back to just `//` comments above the tsdoc comment you add for the release tag. Same with the comment on `baseVersion`.